pkgs/os-specific/linux/iotop-c/default.nix

```
@@ -0,0 +1,28 @@
+{stdenv, fetchFromGitHub, lib, ncurses, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "iotop-c";
+  version = "v1.17";
+
+  src = fetchFromGitHub {
+    owner = "Tomas-M";
+    repo = "iotop";
+    rev = version;
+    sha256 = "0hjy30155c3nijx3jgyn5kpj293632p0j6f3lf5acdfax1ynav86";
+  };
+
+  buildInputs = [ ncurses pkg-config ];
+  buildPhase = "make DESTDIR=$out";
+  installPhase = ''
+    make install DESTDIR=$out
+    ln -s $out/usr/sbin $out/bin
+    mv $out/usr/sbin/iotop $out/usr/sbin/${pname}
+  '';
```

```suggestion
  installFlags = [ "DESTDIR=$out" ];

  postInstall = ''
    mv $out/usr/sbin/iotop $out/bin/iotop-c
  '';
```

nixpkgs usually do not use sbin or usr directories.